snippets/vmware-csi-driver-reqs.adoc

```diff
@@ -0,0 +1,11 @@
+:_content-type: SNIPPET
+
+= VMware vSphere CSI Driver Operator requirements
+
+The following requirements must be met in order to install the CSI Driver Operator:
+
+* VMware vSphere version 6.7U3 or newer
+* Virtual machines of hardware version 15 or newer
+* No third party CSI driver already installed in the cluster
+
+If a third party CSI driver is already present in the cluster, {product-title} will not overwrite it.
```

Comment:
I think I agree with Gregory comment on JIRA. We might want to elaborate on "No third party CSI driver already installed in the cluster" little bit and perhaps also link to https://github.com/openshift/openshift-docs/pull/40571 docs where we specify steps a customer can take to remove the third party CSI driver.